### ISSUE TITLE
Set of labels for different technologies

### ISSUE THREAD

**Issue by toddkarin** [hackingmaterials/pv-terms#26]
We should also standardize the labeling for different technologies, for example, [pvlib.ivtools.fit_sdm_cec_sam](https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.ivtools.fit_sdm_cec_sam.html) uses the list:

- 'monoSi', 'multiSi', 'polySi', 'cis', 'cigs', 'cdte', 'amorphous'

while the cec database [pvlib.pvsystem.retrieve_sam](https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.pvsystem.retrieve_sam.html) uses:

- 'Thin Film', 'Multi-c-Si', 'CIGS', 'Mono-c-Si', 'CdTe'

PVsyst on the other hand uses:
- Si-poly, Si-mono, and others.

Without giving this too much thought, I would probably pick the first option and add this to pvterms. @cwhanse what do you think?

**Comment by steve-ransome** [comment]
Should HIT be added, and either micromorph or else say how many junctions with the amorphous? For clarity how are multi and poly distinguished as the terms have sometimes been used interchangeably.

**Comment by toddkarin** [comment]
The current purpose of these labels is mostly to choose band gaps and material properties for describing the cells, but in the future other applications could be expected. I would say that we want to limit the technology description to descriptions typically available on a datasheet. HIT could be added perhaps as a second level descriptor. Here's an idea:

**Technology level 1**
- mono-Si
- multi-Si
- amorphous-Si
- thin-film
- cigs
- cdte
- other

**Technology level 2** (flexible labels?)
- HIT
- 3 junction
- GaAs

**Comment by cwhanse** [comment]
I'm in favor of agreeing on some string values for cell types.

FYI, in the first two cases @toddkarin lists, pvlib is accommodating terms used by SAM. For `fit_sdm_cec_sam` there's no choice but to use the SAM cell type label, it's passed as a value through the SAM API. For `retrieve_sam`, pvlib is returning the string value that appears in the CEC module database distributed with SAM.

**Comment by toddkarin** [comment]
Yes, I understand. Of course, we can rename cell technology in pvlib when importing from the other libraries. @cwhanse do you have an opinion on the selection and formatting? I think the CEC database naming is a little clunky and would prefer something like what I suggested above.

**Comment by cwhanse** [comment]
I'd suggest avoiding special characters (e.g., no spaces or hyphens), and not starting a value with a numeral if practical. As far as 'monoSi' vs. 'Si-mono', mild preference for the first, but I can see the benefit of the PVsyst style if one is scanning a list of values.

**Comment by toddkarin** [comment]
I would advocate for the hyphen, it makes things a little easier to parse and adds readability: Simono, thinfilm are not that great.

What about capitalization? It's not consistent to have monoSi and cdte.

**Comment by shirubana** [comment]
(Random comment --- I seem to remember HIT is specifically for Panasonic trademarked,  otherwise they have to be called HJT? )
